.github/workflows/jekyll-gh-pages.yml

```diff
@@ -0,0 +1,51 @@
+# Sample workflow for building and deploying a Jekyll site to GitHub Pages
+name: Deploy Jekyll with GitHub Pages dependencies preinstalled
```

Comment:
Sorry if I am unable to understand, but can you help me knowing what's the use of adding this file?

Reply:
@perminder-17 sorry, it was accidental(tried to build it on github using that file)